src/main/java/uk/gov/hmcts/reform/blobrouter/reconciliation/model/out/SuccessfulReconciliationResponse.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.hmcts.reform.blobrouter.reconciliation.model.out;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class SuccessfulReconciliationResponse {
```

Comment:
no reconciliation happened at this stage

Reply:
is it ok to rename the class as `SuccessfulResponse`? or any suggestions?